manifests/kustomize/base/pipeline/postgres/kustomization.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../metadata-writer
+resources:
+  - ml-pipeline-apiserver-deployment.yaml
```

Comment:
Using patches in kustomization.yaml can make this file much simpler: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/

```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
bases:
  - ../metadata-writer
  - ../
patches:
  - ml-pipeline-apiserver-deployment-patch.yaml
```

ml-pipeline-apiserver-deployment-patch.yaml
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: ml-pipeline
spec:
  template:
    spec:
      containers:
        - name: DB_DRIVER_NAME
          valueFrom:
            configMapKeyRef:
              name: pipeline-install-config
              key: dbType
        # PostgreSQL Config
        - name: DBCONFIG_POSTGRESQLCONFIG_USER
          valueFrom:
            secretKeyRef:
              name: postgres-secret
              key: username
        - name: DBCONFIG_POSTGRESQLCONFIG_PASSWORD
          valueFrom:
            secretKeyRef:
              name: postgres-secret
              key: password
        - name: DBCONFIG_POSTGRESQLCONFIG_DBNAME
          valueFrom:
            configMapKeyRef:
              name: pipeline-install-config
              key: pipelineDb
        - name: DBCONFIG_POSTGRESQLCONFIG_HOST
          valueFrom:
            configMapKeyRef:
              name: pipeline-install-config
              key: postgresHost
        - name: DBCONFIG_POSTGRESQLCONFIG_PORT
          valueFrom:
            configMapKeyRef:
              name: pipeline-install-config
              key: postgresPort
        # end of PostgreSQL variables
        name: ml-pipeline-api-server
```

Reply:
They are still slightly different, because `ml-pipeline-apiserver-deployment.yaml` in the main folder also includes some MySQL specific parameters, which are not needed in a PostgreSQL deployment.